src/AppRouter.tsx

```diff
@@ -11,9 +11,14 @@ export const AppRouter = () => {
   return (
     <Switch>
       <Route path="/" exact>
-        {/* TODO: Break into a view component*/}
-        <Blockly />
-        <RobotSimulator />
+        <div className="grid-container">
```

Comment:
I think this is a good time to move this to its own component such as "SimulatorView" or something along those lines. That component can then import the above mentioned CSS file.